components/Layout.tsx

```diff
@@ -22,6 +22,7 @@ export default function Layout({
     <>
       <Head>
         <title>{title ? `${title} - RDocumentation` : 'RDocumentation'}</title>
+        <meta name="google-site-verification" content="l7VjaJvG_OgqxbCyZDDtk_ykMcPXyqwTSoWnf94hGQk" />
```

Comment:
This should be in `_document.tsx`, thats the default file in Next to add those things :+1:

Reply:
Reason why some of these HTML tags are in here instead of all being in `_document.tsx` is because these are dynamic (they change based on the page) vs everything in `_document.tsx` is static

Reply:
I added it to _document instead. I realised after committing that the branch was behind main so I corrected it, thats why there are other files that are changed. should be fine now